build.gradle.kts

```diff
@@ -36,7 +36,7 @@ dependencies {
   implementation("com.google.code.gson:gson:2.8.6")
   compileOnly("org.projectlombok:lombok:1.18.18")
   runtimeOnly("org.hsqldb:hsqldb:2.5.1")
-  runtimeOnly("org.flywaydb:flyway-core:7.5.3")
+  implementation("org.flywaydb:flyway-core:7.5.3")
```

Comment:
Needed by the IntegrationTest class for cleaning the database after each int test